### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by mokutsu**
please review before merging

### CODE REVIEW

File: app/controllers/pledges_controller.rb [hunk 1]
```diff
@@ -10,9 +10,11 @@ def new
   end
 
   def create
-    @pledge = @project.pledges.build(pledge_params)
-    if @pledge.save
+    @reward = Reward.find(params[:reward_id])
```
Comment (joshuarider) [inline app/controllers/pledges_controller.rb:13]: something like if request.xhr? for this branch, and then the normal path for people without JS active?

File: app/assets/javascripts/pledges.js [hunk 2]
```diff
@@ -5,13 +5,28 @@ $(document).on('ready page:load', function(){
...
+
+      if (result) {
+
+        $.ajax({
+          method: 'POST',
+          url: $(this).attr('href'),
+          data: {
+            reward_id: $(this).attr('data')
+          },
+          dataType: 'html',
```
Comment (joshuarider) [inline app/assets/javascripts/pledges.js:20]: redundant at the moment given that we don't want anything back?

File: app/views/projects/_form.html.erb [hunk 3]
```diff
@@ -1,47 +1,50 @@
...
+      <tr>
+        <td><%= f.label :start_time %></td>
+        <td><%= f.datetime_select :start_time %></td>
+      </tr>
+      <tr>
+        <td><%= f.label :end_time %></td>
+        <td><%= f.datetime_select :end_time %></td>
+      </tr>
+    </table>
+    <h2>Rewards</h2>
```
Comment (joshuarider) [inline app/views/projects/_form.html.erb:28]: div up the whole rewards section so we have a target for our reward form append?

File: app/controllers/pledges_controller.rb (hunk 1 above)
Comment (mokutsu) [inline app/controllers/pledges_controller.rb:13]: sounds good! graceful degradation! Not sure of the difference between request.xhr and specifying json or javaScript.

File: app/assets/javascripts/pledges.js (hunk 2 above)
Comment (mokutsu) [inline app/assets/javascripts/pledges.js:20]: not sure what is redundant here, we still need to make a post request with a confirm?

Comment (mokutsu) [inline app/assets/javascripts/pledges.js:20]: I think doing a pop up modal (without leaving space for it makes sense). This way, we can worry less about screen size and we might be able to be a bit less exact with positioning

File: app/assets/javascripts/pledges.js [hunk 4]
```diff
@@ -5,13 +5,28 @@ $(document).on('ready page:load', function(){
...
+          success: function(returned_info){
+            console.log('ok, post succeded.');
+
+            $('.modal-thanks').animate({
+              opacity: 1}, 500).delay(10000).animate({opacity: 0}, 500);
```
Comment (mokutsu) [inline app/assets/javascripts/pledges.js:25]: I think doing a pop up modal (without leaving space for it makes sense). This way, we can worry less about screen size and we might be able to be a bit less exact with positioning

File: app/assets/stylesheets/pledges.scss [hunk 5]
```diff
@@ -1,3 +1,14 @@
 // Place all the styles related to the Pledges controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.modal-thanks {
```
Comment (mokutsu) [inline app/assets/stylesheets/pledges.scss:5]: let's style with bs (boot strap)

File: app/assets/stylesheets/projects.scss [hunk 6]
```diff
@@ -1,3 +1,63 @@
 // Place all the styles related to the Projects controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+* {
+  margin: 0;
```
Comment (mokutsu) [inline app/assets/stylesheets/projects.scss:6]: i suggest we comment all this out and start pulling things in by bs selector

**Pull request merged**